Source/WebCore/html/HTMLIFrameElement.cpp

```diff
@@ -190,6 +191,22 @@ void HTMLIFrameElement::setLoadingForBindings(const AtomString& value)
     setAttributeWithoutSynchronization(loadingAttr, value);
 }
 
+String HTMLIFrameElement::srcdoc() const
+{
+    return getAttribute(srcdocAttr);
```

Comment:
This should be able to use `fastGetAttribute`, if I remember the name correctly.

Reply:
I can't seem to find a fastGetAttribute or anything with a similar name?

Reply:
Sorry, long ago it was renamed to `attributeWithoutSynchronization`.